apps/dashboard/src/components/welcome/ai-prompts/simple-prompt-getter.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ANGULAR_PROMPT,
+  JAVASCRIPT_PROMPT,
+  NEXTJS_PROMPT,
+  REACT_NATIVE_PROMPT,
+  REACT_PROMPT,
+  REMIX_PROMPT,
+  VUE_PROMPT,
+} from './framework-prompts';
+
+export function getFrameworkPrompt(
+  frameworkName: string,
+  applicationIdentifier?: string,
+  region: 'us' | 'eu' = 'us'
+): string {
+  const regionConfig = {
+    eu: {
+      socketUrl: 'wss://eu.ws.novu.co',
+      backendUrl: 'https://eu.api.novu.co',
```

Comment:
You should be able to get those from the global config (API_ROOT and WEBSOCKET ROOT)